src/Workspaces/Core/Portable/Workspace/Solution/ProjectInfo.cs

```diff
@@ -497,7 +496,8 @@ internal sealed class ProjectAttributes(
             public Guid TelemetryId { get; } = telemetryId;
 
             private StrongBox<(string?, string?)>? _lazyNameAndFlavor;
```

Comment:
Should we also transition this other field over too? (oddly it's doing a different pattern...)

Reply:
Sure.  but let me do in followup pr, ok? :)

Reply:
Sounds good!